cmd/api/api.go

```diff
@@ -244,19 +245,19 @@ func (c *command) authMiddleware(next http.Handler, usage string) http.Handler {
 	return http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
 		auth := r.Header.Get("Authorization")
 		if auth == "" {
-			sendError(fmt.Errorf("go away"), w, http.StatusUnauthorized)
+			sendError(errors.New("go away"), w, http.StatusUnauthorized)
 			return
 		}
 
 		parts := strings.Split(auth, "Bearer ")
 		if len(parts) == 2 {
 			token := parts[1]
 			if !c.isValidToken(r.Context(), token, usage) {
-				sendError(fmt.Errorf("go away"), w, http.StatusUnauthorized)
+				sendError(errors.New("go away"), w, http.StatusUnauthorized)
 				return
 			}
 		} else {
-			sendError(fmt.Errorf("go away"), w, http.StatusUnauthorized)
+			sendError(errors.New("go away"), w, http.StatusUnauthorized)
```

Comment:
Maybe this `go away` error warrants a `var ErrUnauthorized = errors.New("go away")`.